### DISCUSSION TITLE
PAGES ERROR 404

### DISCUSSION THREAD

**Discussion by MwiseGloria** [community/community#52209] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

![image](https://user-images.githubusercontent.com/129226307/230503246-c3534ac9-d25d-444e-8db4-e47b7c6a1e09.png)

I keep having this issue when trying to create a page,  anybody know how to fix it?

**Comment by radar-blip** [comment]
It's because your `index.html` file is inside a folder and GitHub Pages cant see it. 
If you put it outside the folder (root) it should work.

Note: you may have to adjust some links accordingly.

**Reply by MwiseGloria** [reply]
Thank you so much David, it is working now

**Reply by radar-blip** [reply]
You're welcome!
